chapters/unitexpressions.tex

```diff
@@ -65,10 +65,24 @@ \section{The Syntax of Unit Expressions}\label{the-syntax-of-unit-expressions}
    NON-DIGIT
 \end{lstlisting}
 
-It is required that basic and derived units of the SI system are recognized, but tools are allowed to additionally support user-defined unit symbols.
+It is required that basic and derived units of the SI system are recognized and in addition some units compatible with the SI system listed below, but tools are allowed to additionally support user-defined unit symbols.
 The required unit symbols do not make use of Greek letters, but a unit such as $\Omega$ is spelled out as \lstinline!"Ohm"!.
 Similarly degree is spelled out as \lstinline!"deg"!, both on its own (for angles) and as part of \lstinline!"degC"!, \lstinline!"degF"! and \lstinline!"degRk"! for temperatures (Celsius, Fahrenheit and Rankine).
 
+It is recommended that non SI units are only used for the displayUnit attribute -- that ensures that the model is valid even if the unit is not recognized.
+
+The units compatible with the SI system that should be recognized are:
+\begin{itemize}
+\item minute "min" (1 minute=60 s)
+\item hour "h" (1 hour=3600 s)
+\item day "d" (1 day=86400 s)
+\item litre "l" and "L" (1 litre=1 dm3)
+\item electronVolt "eV" (1 electronvolt=1.602176634e-19 J)
```

Comment:
```suggestion
\item electron volt "eV" (1 electron volt=1.602176634e-19 J)
```
or `electronvolt`?

Reply:
The SI Brochure calls it "electronvolt" so I assume that should be used.
```suggestion
\item electronvolt "eV" (1 electronvolt=1.602176634e-19 J)
```